crates/bevy_core/src/time/time.rs

```diff
@@ -2,6 +2,52 @@ use bevy_ecs::system::ResMut;
 use bevy_utils::{Duration, Instant};
 
 /// Tracks elapsed time since the last update and since the App has started
+///
+/// # Examples
```

Comment:
I love this doc test: this is incredibly helpful.

Reply:
Thanks!